linkerd/app/outbound/src/lib.rs

```diff
@@ -406,18 +484,23 @@ impl Config {
             + 'static,
         S::Future: Send,
     {
+        let prevent_loop = PreventLoop::from(listen_addr.port());
+
+        // Load balances TCP streams that cannot be decoded as HTTP.
+        let tcp_balance =
+            svc::stack(self.build_tcp_balance(&tcp_connect, resolve, prevent_loop, &metrics))
+                .push_map_target(|a: listen::Addrs| a.target_addr())
+                .into_inner();
+
```

Comment:
it's above the `ProxyConfig` binding because the `ProxyConfig` binding moves out of `self`, and `build_tcp_balance` borrows `self`. we could probably change the `ProxyConfig` binding to be by-ref though.